### ISSUE TITLE
Allow left arrow to foldup/rollup parent directory.

### ISSUE THREAD

**Issue by brandonjp** [iterate-ch/cyberduck#12660]
**Is your feature request related to a problem? Please describe.**
On MacOS in the Finder, I often navigate folders with the keyboard arrows.  When I move over to Cyberduck and navigate into a folder using the right arrow, I regularly press the left arrow to go back up to the parent directory.  In Finder this works. In Cyberduck it does not.  

**Describe the solution you'd like**
In Cyberduck, when inside a folder, when I expand a sub folder, I'd like to be able to press the back arrow to return to the parent and rollup/foldup the folder.  This would then move the highlight to the containing folder you were just browsing.

**Describe alternatives you've considered**
I am aware that you can click and/or scroll to accomplish his, however when navigating a huge list of files, pressing the left arrow would remove the need to move away from the keyboard in situations such as this.

**Additional context**
Here is how the left and right arrows behave in Finder: https://cln.sh/PkPvyu

**Comment by dkocher** [comment]
The only difference I see is that pressing left arrow does not move the selection from a currently selected item in the folder to the folder allowing a subsequent press of left arrow to close it. However if you move the current selection first to the folder using the up arrow keys you should be able to close the expanded folder just fine.

**Comment by brandonjp** [comment]
Yeah, that's correct. The UX benefit of allowing a left arrow to move selection to the parent folder is a huge benefit.  Prevents having to touch the mouse or having to press the up arrow hundreds of times if you're in a long list of files.

**Comment by DJF3** [comment]
@brandonjp   I completely agree.  Often I have very long lists of files and it would be fantastic if you could "arrow-left" to instantly move to the parent-folder instead of having to scroll up or click using the mouse. 
The "arrow-left-for-parent-folder" is just like Macos Finder works.

**Comment by jaredko-po** [comment]
I'd love to add my +1 to this issue. It's a UI behavior that's consistent across all other apps (even those not created by Apple). I could be 30 files down and instinctively do the left-arrow key to close that branch with nothing happening. Super minor issue, but still a frustrating user experience.

**Comment by brandonjp** [comment]
Yea. Seems so small but I literally stopped using cyberduck because of this. Super inconvenient when you’re in a folder of 2000 files￼ (or even 200).

Sent with Shortwave <https://www.shortwave.com?utm_medium=email&utm_content=signature&utm_source=YnJhbmRvbmpwQGdtYWlsLmNvbQ==>

On Fri Apr 3, 2026, 10:10 PM GMT, Jared Ko ***@***.***> wrote:
[image]jaredko-po left a comment (iterate-ch/cyberduck#12660) <https://github.com/iterate-ch/cyberduck/issues/12660#issuecomment-4185408404>
